cli/cmd/project/edit.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func EditCmd(ch *cmdutil.Helper) *cobra.Command {
 	var name, description, prodVersion, prodBranch, subpath, path, provisioner string
-	var public bool
+	var flagSet, public bool
```

Comment:
Same comment as earlier, but notice how this makes it look like `flagSet` is populated from a flag (instead of being internal state)?